cytominer_eval/tests/test_operations/test_precision_recall.py

```diff
@@ -42,11 +42,11 @@ def test_precision_recall():
     result = precision_recall(
         similarity_melted_df=similarity_melted_df,
         replicate_groups=replicate_groups,
-        k=10,
+        k=[5, 10],
```

Comment:
can you also keep a test for k as an `int`?

This test is good, but maybe add a test asserting that the outputs are the same for `k=10` and `k=[10]`

Reply:
done